runners/google-cloud-dataflow-java/build.gradle

```diff
@@ -37,8 +37,8 @@ evaluationDependsOn(":beam-sdks-java-container")
 
 processResources {
   filter org.apache.tools.ant.filters.ReplaceTokens, tokens: [
-    'dataflow.legacy_environment_major_version' : '7',
-    'dataflow.fnapi_environment_major_version' : '7',
+    'dataflow.legacy_environment_major_version' : '8',
```

Comment:
Upping the environment major version is only needed if you plan to fix the way that legacy expansions worked.